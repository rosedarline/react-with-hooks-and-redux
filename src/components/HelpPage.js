import React from "react";

function HelpPage () {
    return (
        <div>This is from my help component</div>
    );
} 

export default HelpPage;