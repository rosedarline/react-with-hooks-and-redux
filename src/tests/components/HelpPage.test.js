import React from "react";
import { shallow } from "enzyme";
import HelpPage from "../../components/HelpPage";

test("Should render HelpPage correctly", () => {
    const wrapper = shallow(<HelpPage />);
    expect(wrapper).toMatchSnapshot();
});