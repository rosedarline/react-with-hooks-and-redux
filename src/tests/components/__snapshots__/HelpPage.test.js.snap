// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render HelpPage correctly 1`] = `
<div>
  This is from my help component
</div>
`;
